invenio_rdm_records/services/schemas/record.py

```diff
@@ -88,3 +90,37 @@ def default_nested(self, data, many, **kwargs):
             data["custom_fields"] = {}
 
         return data
+
+    def hide_tombstone_or_metadata(self, data):
+        """Hide tombstone info if the record isn't deleted and metadata if it is."""
+        if not data.get("deletion_status", {}).get("is_deleted", False):
```

Comment:
note that `deletion_status.is_deleted` is different from the top-level `is_deleted` (which has been around forever already)